app/src/main/java/com/edulectronics/tinycircuit/Controllers/DrawingController.java

```diff
@@ -0,0 +1,53 @@
+package com.edulectronics.tinycircuit.Controllers;
+
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.Point;
+
+import com.edulectronics.tinycircuit.Models.Components.Connectors.ConnectionPoint;
+import com.edulectronics.tinycircuit.Models.Components.Connectors.ConnectionPointOrientation;
+import com.edulectronics.tinycircuit.R;
+import com.edulectronics.tinycircuit.Views.DrawView;
+
+/**
+ * Created by bernd on 09/12/2016.
+ */
+
+public class DrawingController {
```

Comment:
Naamgeving niet goed. Deze controller tekent niks. Het doet ook niets met models/views dus is geen controller. Misschien CoordinateHelper of iets dergelijks?